code/modules/supermatter/supermatter.dm

```diff
@@ -193,16 +194,14 @@
 			speech.name = "Supermatter [short_name] Monitor"
 			speech.job = "Automated Announcement"
 			speech.as_name = "Supermatter [short_name] Monitor"
-			Broadcast_Message(speech, level = list(0,1))
+			Broadcast_Message(speech, level = current_zlevel)
```

Comment:
So level = list(current_zlevel)?

Makes sense, seemed a bit shaky when testing it